packages/nimble-components/src/table-column/mapping/mappings/text/tests/mapping-text.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    usageWarning
+} from '../../../../../utilities/tests/storybook';
+
+const metadata: Meta<MappingTextArgs> = {
+    title: 'Table Column Types'
+};
+
+export default metadata;
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface MappingTextArgs {}
+
+export const textMapping: StoryObj<MappingTextArgs> = {
+    parameters: {
+        docs: {
+            description: {
+                story: 'The `nimble-mapping-text` element defines a mapping from a data value to display text. It is meant to be used as content of the `nimble-table-column-mapping` column type element.'
+            }
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<MappingTextArgs>`
+        ${usageWarning('table')}
```

Comment:
These mapping elements don't render anything, but we still want to document them, so I created stories for them.

Reply:
Can you brainstorm some other approaches for how this should be documented? Your sentiment is right that we should document it. If you come in with fresh eyes as one of our clients and see the options organized this way is that going to be the clearest for you?

![image](https://github.com/ni/nimble/assets/1588923/37932371-1369-4430-a76f-d0b9857e5b31)

Does seeing the story organized this way with nothing rendering feel inline with our other elements?

![image](https://github.com/ni/nimble/assets/1588923/8f75152b-a363-4076-b24d-3045e3b6ea41)

I'm not sure if you wanted to brainstorm together or were looking for specific feedback or discussion since there isn't any open comments asking for feedback. So I'll state it seems confusing and ask if you can come up with some clearer options.

Reply:
Sorry, I reset you but fogot this was still open. I'm still thinking about how we can do this differently.

Reply:
I've taken a pass at it. I did the following:
- Broke up table column types into their own top-level components, named like "Table Column - Text" with the hyphen so that "Table Column Configuration" is not mistaken for a column type.
- Created .mdx files for the Icon and Enum Text column types so that they can pull in documentation about the mapping elements
- Moved the mapping elements to an "Ignore" folder which is ordered last in the list. There was no proper way to hide these stories.

Reply:
For future reference, here's the Storybook issue requesting the ability to hide a story. It contains a few workarounds but I agree they are all flawed to varying degrees.

https://github.com/storybookjs/storybook/issues/9209

Reply:
An earlier change (removing Mappings from column config) broke everything. I've reworked things and everything is working again.

Reply:
New day new opinion 😇: after playing with the storybook I think the "Ignore" section is more confusing than helpful. I see "Ignore" and I can't stop myself from NOT ignoring it!

My new suggestion would be to move the mapping elements under their corresponding column sections (in storybook, not in the source tree). If we think blank pages would be confusing then maybe we could set their contents to be some placeholder text like "This element only renders when it is a child of the XXX column. See the Docs page for more information."

You're welcome to wait for more opinions before implementing a new direction (or sticking with the current direction).

Reply:
Actually, the mapping elements _never_ render. Would it feel better to rename the "Ignore" component to "Non-rendered elements"? That might be less tempting/mysterious than "Ignore". I think it's better to hide/downplay those stories, because they're really not useful for anything given that we're already pulling their docs into another page.

Reply:
I can convince myself that Tests is a good spot: we're testing that the element doesn't render anything, just like the other "intentionally blank" tests prove that the `hidden` attribute works.

Reply:
Moved